### ISSUE TITLE
"New Project" defaults to personal GitHub instead of organization account

### ISSUE THREAD

**Issue by rickroesler** [forcedotcom/devops-center-feedback#174]
**Describe the bug**
"New Project" is defaulting to my personal GitHub rather then the organization account

**To Reproduce**
1. authorize the two **Salesforce Integration Applications** in the organization account per [instructions](https://help.salesforce.com/s/articleView?id=sf.devops_center_org_owns_github_repo.htm&type=5)
2. delete the "DevOps Center GitHub" **Authentication Setting** per [instructions](https://help.salesforce.com/s/articleView?id=sf.devops_center_org_owns_github_repo.htm&type=5)
3. Open DevOps Center
4. Select **New Project**
5. you're forced to re-authenticate with GitHub (and the DevOps Center GitHub **Authentication Setting** is re-created)
6. Select **New Project**
7. The "Repository Owner" defaults to personal

**Expected behavior**
Default to organization account

**Comment by RupertBarrow** [comment]
I had the same issue as you, but all the steps you mentioned worked fine for me.

**Comment by rickroesler** [comment]
@RupertBarrow All the steps work fine for me, too. Except that I expected the Repository Owner to default to the organization; I thought that was the whole point of the exercise.

**Comment by rickroesler** [comment]
@geomuck This question is feeling very lonely. Just need to know if this is the expected behaviour.

**Comment by franciscoperezsammartino** [comment]
Hi @rickroesler , were you able to solve this issue. Sorry for the delay in responding.

Thanks very much.

**Comment by rickroesler** [comment]
@franciscoperezsammartino No. Mostly just need to know if this is the way it's intended to work (as described above). If it is, recommend that it be changed, if possible, to default to the GitHub organization. If it's not, then something else is wrong.
